mobile/src/main/java/org/openhab/habdroid/util/AsyncHttpClient.java

```diff
@@ -78,19 +77,52 @@ public Bitmap convertBodyInBackground(ResponseBody body) throws IOException {
             }
         }
 
-        private Bitmap getBitmapFromSvgInputstream(InputStream is) throws SVGParseException {
+        private static Bitmap getBitmapFromSvgInputstream(Resources res, InputStream is, int size)
+                throws SVGParseException {
             SVG svg = SVG.getFromInputStream(is);
-            RectF viewBox = svg.getDocumentViewBox();
-            double width = viewBox != null ? viewBox.width() : mDefaultSize;
-            double height = viewBox != null ? viewBox.height() : mDefaultSize;
-
+            svg.setRenderDPI(DisplayMetrics.DENSITY_DEFAULT);
+            Float density = res.getDisplayMetrics().density;
```

Comment:
Float -> float

Reply:
density might be set to null below

Reply:
Oops, right...didn't remember my own code anymore :-/